.github/workflows/win.yml

```diff
@@ -292,6 +292,25 @@ jobs:
           cp libs/* crystal/lib/
           cp src/* crystal/src -Recurse
           rm crystal/src/llvm/ext/llvm_ext.obj
+          echo "$(pwd)\crystal" >> $env:GITHUB_PATH
+
+      - name: Get latest shards release
+        run: |
+          echo "SHARDS_RELEASE_TAG=$(Invoke-RestMethod https://api.github.com/repos/crystal-lang/shards/releases/latest | Select-Object -ExpandProperty tag_name)" >> ${env:GITHUB_ENV}
```

Comment:
The shards release should be pinned to a specific version. We'll add that to the checklist for cutting a shards release (https://github.com/crystal-lang/distribution-scripts/blob/master/processes/shards-release.md). It's safer to have an explicit version.

Reply:
I've snapped it to v0.16.0.